estimagic/dashboard/convergence_plot.py

```diff
@@ -0,0 +1,157 @@
+"""Functions for creating and styling the convergence plot."""
+import random
+from functools import partial
+
+import bokeh.palettes
+from bokeh.models import ColumnDataSource
+from bokeh.plotting import figure
+from tornado import gen
+
+
+def setup_convergence_plot(param_df, initial_fitness):
```

Comment:
This should be called setup_convergence_plots (plural) or setup_convergence_tab, depending on which is more appropriate. I think I prefer the tab.

Reply:
Instead of using plural we will split up the functions.